platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/DefaultConfigurationCacheIO.kt

```diff
@@ -65,23 +68,24 @@ import java.io.OutputStream
 
 
 @ServiceScope(Scope.Build::class)
-class ConfigurationCacheIO internal constructor(
+internal
+class DefaultConfigurationCacheIO internal constructor(
```

Comment:
ℹ️  There is no need for the `ConfigurationCacheIO` interface, because each consumer actually cares only about certain IO functionality now represented by more fine-grained interfaces